requires/development.txt

```diff
@@ -0,0 +1,6 @@
+-r requires/installation.txt
```

Comment:
Use `-e .` for dev reqs. Amongst other things that don't really apply to this project, it ensures deps as-seen-by `setup.py` are used. It's already being done in `.travis.yml` too.

Reply:
Added